lib/scaleway/charts/q-application/templates/secret.j2.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ sanitized_name }}
+  namespace: {{ namespace }}
+  labels:
+    ownerId: {{ owner_id }}
+    envId: {{ environment_id }}
+    appId: {{ id }}
+    app: {{ sanitized_name }}
+type: Opaque
+stringData:
+  {%- for ev in environment_variables %}
+  {{ ev.key }}: "{{ ev.value }}"
```

Comment:
Use raw string please.
Take a look at the latest change in DO & AWS for that

Reply:
Done